scripts/check_secrets.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2018 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import os
+import sys
+
+import hypothesistooling as tools
+
+if __name__ == '__main__':
+    if os.environ.get('TRAVIS_SECURE_ENV_VARS', None) != 'true':
+        sys.exit(0)
+
+    tools.decrypt_secrets()
```

Comment:
Maybe add an assert that this does something useful? e.g.

```python
assert not os.path.exists(tools.SECRETS_BASE)
tools.decrypt_secrets()
assert os.path.exists(tools.SECRETS_BASE)
```

Reply:
SECRETS_BASE actually doesn't exist - we unpack everything into the root (this is probably a bad idea, but fixing it is out of scope for this PR... I'll look into more sensible secret management when I merge in hypothesis-ruby)